Examples/rc-maestro/rc-maestro/Sources/rc-maestro/RcMaestroApp.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+import RevenueCat
+
+@main
+struct RcMaestroApp: App {
+
+    init() {
+        Purchases.logLevel = .debug
+        Purchases.configure(withAPIKey: "appl_fEVHDkWYraujHYbxopxJknYGNUx")
```

Comment:
Commited API key alert! 😄  Perhaps use a xcconfig file?

Reply:
classic

Reply:
I've followed the same approach of the workspace.